[2025-09-06T16:37:24.520+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-06T16:37:24.558+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_transformations.wait_for_load manual__2025-09-06T16:37:18.863576+00:00 [queued]>
[2025-09-06T16:37:24.564+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_transformations.wait_for_load manual__2025-09-06T16:37:18.863576+00:00 [queued]>
[2025-09-06T16:37:24.564+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-06T16:37:24.572+0000] {taskinstance.py:2330} INFO - Executing <Task(ExternalTaskSensor): wait_for_load> on 2025-09-06 16:37:18.863576+00:00
[2025-09-06T16:37:24.578+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=4219) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-06T16:37:24.580+0000] {standard_task_runner.py:64} INFO - Started process 4221 to run task
[2025-09-06T16:37:24.578+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_transformations', 'wait_for_load', 'manual__2025-09-06T16:37:18.863576+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/dbt_transformations.py', '--cfg-path', '/tmp/tmph_rqvuku']
[2025-09-06T16:37:24.581+0000] {standard_task_runner.py:91} INFO - Job 83: Subtask wait_for_load
[2025-09-06T16:37:24.613+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_transformations.wait_for_load manual__2025-09-06T16:37:18.863576+00:00 [running]> on host dd8e0e853e24
[2025-09-06T16:37:24.675+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_transformations' AIRFLOW_CTX_TASK_ID='wait_for_load' AIRFLOW_CTX_EXECUTION_DATE='2025-09-06T16:37:18.863576+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-06T16:37:18.863576+00:00'
[2025-09-06T16:37:24.676+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-06T16:37:24.688+0000] {baseoperator.py:400} WARNING - ExternalTaskSensor.execute cannot be called outside TaskInstance!
[2025-09-06T16:37:24.695+0000] {external_task.py:258} INFO - Poking for tasks ['print_success'] in dag load_customers_to_postgres on 2025-09-06T16:37:18.863576+00:00 ... 
[2025-09-06T16:37:24.702+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/***/dags/load_customers_to_postgres.py
[2025-09-06T16:37:24.720+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/session.py:79 DeprecationWarning: This method is deprecated and will be removed in future.
[2025-09-06T16:37:24.728+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-06T16:37:24.739+0000] {taskinstance.py:2865} INFO - Rescheduling task, marking task as UP_FOR_RESCHEDULE
[2025-09-06T16:37:24.760+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-06T16:37:24.771+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-06T16:37:24.772+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
